QUICKSTART.md

```diff
@@ -72,12 +72,16 @@ Create the first edge cluster:
 kind create cluster --name florin --config  kubestellar/examples/florin-config.yaml
 ```  
 
+Note: if you already have a cluster named 'florin' from a previous installation of KubeStellar, please delete the florin cluster ('kind cluster delete florin') and create it using the instruction above.
```

Comment:
Those kind clusters are from the example usage, not the KubeStellar installation.  How about s/installation of KubeStellear/exercise of KubeStellar/ ?